From ajay.gupta@ti.com  Tue May  4 12:19:59 2010
From: Ajay Kumar Gupta <ajay.gupta@ti.com>
Date: Tue, 4 May 2010 19:53:09 +0530
Subject: usb: ehci-omap: fix compilation warning
Cc: "felipe.balbi@nokia.com" <felipe.balbi@nokia.com>
Message-ID: <19F8576C6E063C45BE387C64729E7394044E351733@dbde02.ent.ti.com>


Fixes below compilation warning:
drivers/usb/host/ehci-hcd.c:425:
	warning: 'ehci_port_power' defined but not used

Signed-off-by: Ajay Kumar Gupta <ajay.gupta@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-omap.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/usb/host/ehci-omap.c
+++ b/drivers/usb/host/ehci-omap.c
@@ -659,6 +659,9 @@ static int ehci_hcd_omap_probe(struct pl
 		goto err_add_hcd;
 	}
 
+	/* root ports should always stay powered */
+	ehci_port_power(omap->ehci, 1);
+
 	return 0;
 
 err_add_hcd:
